actors/migration/nv15/top.go

```diff
@@ -57,6 +53,20 @@ type MigrationCache interface {
 	Load(key string, loadFunc func() (cid.Cid, error)) (cid.Cid, error)
 }
 
+func CidKey(c cid.Cid) string {
+	return c.String()
+}
+
+func MinerPrevSectorsInKey(m address.Address) string {
```

Comment:
we'll likely wanna persist these to allow for restarts after premigrations have run. maybe.

Reply:
This is a lotus side thing but I agree